File: app/src/main/java/com/zetapush/testsdk/NoCredentialsStorage.java
package com.zetapush.testsdk;

import java.util.Map;

/**
 * Created by damien on 24/07/17.
 */

public class NoCredentialsStorage implements StorageCredentialsInterface {
    @Override
    public void saveCredentials(String login, String password) {
        // Do nothing
    }

    @Override
    public Map<String, String> getCredentials() {
        return null;
    }

    @Override
    public void clearCredentials() {
        // Do nothing
    }
}

File: android-library/src/main/java/com/zetapush/library/NoCredentialsStorage.kt
package com.zetapush.library

/**
 * Created by damien on 24/07/17.
 */

class NoCredentialsStorage : StorageCredentialsInterface {
    override fun saveCredentials(login: String, password: String) {
        // Do nothing
    }

    override fun getCredentials(): Map<String, String>? {
        return null
    }

    override fun clearCredentials() {
        // Do nothing
    }
}
